api/v1/address.go

```diff
@@ -18,11 +18,7 @@ type AddressResponse struct {
 	AddressIsSigning bool   `json:"is_signing" gorm:"default:true;not null"`
```

Comment:
These changes will disappear from this pr once we agree on how to do the db passing in the address pr